authors/settings.py

```diff
@@ -153,7 +153,7 @@
         'authors.apps.authentication.backends.JWTAuthentication',
     ),
     'DEFAULT_PAGINATION_CLASS': 'rest_framework.pagination.LimitOffsetPagination',
-    'PAGE_SIZE': 10,
+    'PAGE_SIZE': 2,
```

Comment:
I suggest increasing the page size to five or above

Reply:
I have implemented that.